### ISSUE TITLE
Install from debian packages instead of using the netdata installer

### ISSUE THREAD

**Issue by arthurlogilab** [YunoHost-Apps/netdata_ynh#15]
Netdata is packaged for debian : https://packages.debian.org/search?keywords=netdata 

* stretch-backports: 1.6
* buster : 1.8
* sid: 1.8

When yunohost gets to stretch, this would be nice (unless a jessie-backport could be hosted somewhere).

**Comment by JimboJoe** [comment]
Looks like a good idea, but provided Debian lifecycle, wouldn't it be better to be able to use "bleeding edge" versions (I do think so)?

**Comment by arthurlogilab** [comment]
I agree with you, but I think this is a good example of software where yunohost would benefit from either pushing those packages through the official backports route, or providing their own infrastructure for backports to have more recent versions of already packaged software.

**Comment by alexAubin** [comment]
Issue is 5 years old, closing as wontfix
